app/build.gradle

```diff
@@ -63,7 +63,8 @@ dependencies {
     debugImplementation "androidx.compose.ui:ui-test-manifest:$compose_ui_version"
 
     // Add Room Database Dependencies
-    implementation "androidx.room:room-runtime:2.5.0"
-    kapt "androidx.room:room-compiler:2.5.0"
-    implementation "androidx.room:room-ktx:2.5.0"
+    def room_version = "2.5.0"
+    implementation "androidx.room:room-runtime:$room_version"
```

Comment:
Added version variable to make it a little easier to update any room dependencies.

Reply:
Good idea.  Dependencies are like whack a mole in Android.